packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -84,6 +84,7 @@ class ProjectGet(OutputSchema):
     dev: dict | None
     permalink: ProjectPermalink = FieldNotRequired()
     workspace_id: WorkspaceID | None
+    folder_id: FolderID | None
```

Comment:
Yes current we have 2 concepts/resources
 - **workspaces**
   - If NULL it means it is User private workspace
   - If not NULL it is a shared workspace (Yes you can make abstraction that it can be viewed as a mount)
-  **folders**
  -  if NULL it is a root folder
  -  if not NULL it is a normal folder with ID
  - NOTE: Folders can live in both Private but also Shared workspaces